### PR TITLE
5963 create ui play music

### PR DESCRIPTION
**Pull request by Namvn97**
https://edu-redmine.framgia.vn/issues/5963?issue_count=18&issue_position=12&next_issue_id=5964&prev_issue_id=5962

create ui play music

### CODE REVIEW

**Comment by Namvn97** [comment]
ready

File: app/src/main/java/framgia/phannam/android/nmusic/ui/MainActivity.java [hunk 1]
```diff
@@ -13,8 +13,10 @@
 import com.fristproject.android.music55.R;
 
 import framgia.phannam.android.nmusic.ui.home.FragmentGenres;
+import framgia.phannam.android.nmusic.ui.playmusic.PlayActivity;
 import framgia.phannam.android.nmusic.ui.today.FragmentToDay;
 
+
 public class MainActivity extends AppCompatActivity implements
         BottomNavigationView.OnNavigationItemSelectedListener {
     public static final String TITLE_ACTIONBAR = "Music";
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/phannam/android/nmusic/ui/MainActivity.java]:
```suggestion
    public static final String TITLE_ACTION_BAR = "Music";
```

File: app/src/main/java/framgia/phannam/android/nmusic/ui/MainActivity.java [hunk 2]
```diff
@@ -47,9 +49,9 @@ private void loadFragment(Fragment fragment) {
         transaction.commit();
     }
 
-    private void loadFragmentToDay(Fragment fragment){
-        FragmentTransaction transaction1 =getSupportFragmentManager().beginTransaction();
-        transaction1.add(R.id.frame_today,fragment);
+    private void loadFragmentToDay(Fragment fragment) {
+        FragmentTransaction transaction1 = getSupportFragmentManager().beginTransaction();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/phannam/android/nmusic/ui/MainActivity.java]:
```suggestion
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
```

File: app/src/main/java/framgia/phannam/android/nmusic/ui/playmusic/PlayActivity.java [hunk 3]
```diff
@@ -1,8 +1,66 @@
...
+import com.fristproject.android.music55.R;
+
 /**
  * Created by namp5 on 1/17/2019.
  */
 
-public class PlayActivity {
+public class PlayActivity extends AppCompatActivity {
+    private Toolbar mToolbar;
+    public static final String TITLE_ACTIONBAR = "";
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/phannam/android/nmusic/ui/playmusic/PlayActivity.java]:
```suggestion
    public static final String TITLE_ACTION_BAR = "";
```

File: app/src/main/java/framgia/phannam/android/nmusic/ui/home/FragmentGenres.java [hunk 4]
```diff
@@ -22,45 +22,46 @@
...
-        mCategoriesAdapter.setGenres(genres);
-    }
+            }
 
-    @Override
-    public void showSongsByGenres() {
-    }
-}
+            @Override
+            public void showGenres (List < Genres > genres) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/phannam/android/nmusic/ui/home/FragmentGenres.java]: format code

File: app/src/main/java/framgia/phannam/android/nmusic/ui/playmusic/PlayActivity.java [hunk 5]
```diff
@@ -1,8 +1,66 @@
...
+        initView();
+
+    }
+
+    private void initView() {
+        mToolbar = findViewById(R.id.tool_bar_play_music);
+        setSupportActionBar(mToolbar);
+        getSupportActionBar().setTitle(TITLE_ACTIONBAR);
+        mToolbar.setNavigationIcon(R.drawable.ic_back);
+        mToolbar.setNavigationOnClickListener(new View.OnClickListener() {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/phannam/android/nmusic/ui/playmusic/PlayActivity.java]: implement thay vì tạo new

File: app/src/main/res/layout/activity_play_music.xml [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/text_name_song_play_music" />
+
+    <ImageView
+        android:id="@+id/image_song_play_music"
+        android:layout_width="@dimen/dp_250"
+        android:layout_height="@dimen/dp_190"
+        android:layout_marginTop="@dimen/dp_32"
+        android:scaleType="centerCrop"
+        android:src="@drawable/playmusic"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_play_music.xml]: convention

**Comment by Namvn97** [comment]
ready

**Comment by Namvn97** [comment]
ready

**Pull request merged**